README.md

```diff
@@ -47,9 +47,8 @@ The file ReadMe.txt in a build bundle describes how to install the library on th
    14. [Thread Safety](#2-14)
    15. [Library Initialisation and Cleanup](#2-15)
    16. [Cross Platform Build and Test on Linux for MinGW](#2-16)
-   17. [Building with C++11 and other compilers](#2-17)
-   18. [Static and Shared Libraries](#2-18)
-   19. [Support for bmff files (CR3, HEIF, HEIC, and AVIF)](#2-19)
+   17. [Static and Shared Libraries](#2-18)
```

Comment:
I find a bit problematic to use hardcoded numbers for creating this list (and the links to the different sections). 

By removing the section "Building with C++11 and other compilers", I would need to update the numbers of the consecutive sections (2-18 --> 2-17;  2-19 --> 2-18).

I think it would make sense to change the name of the sections from numerical to string format. For example:
```
<div id="2-19">
# --> 
<div id="StaticAndSharedLibs">
```

Reply:
That's fine, @piponazo.  I'm not going to do that on the 0.27-maintenance branch.  I hope 0.27-maintenance will soon be history!

I started using numbers because printed books use them (so the user can navigate).  Markdown is much better than a printed book because it can have links.  So, you can focus on content.